contrib/gtwvg/paint.prg

```diff
@@ -840,7 +840,10 @@ FUNCTION wvg_LoadIcon( ncIcon )
 
    RETURN wapi_LoadImage( , ncIcon, WIN_IMAGE_ICON,,, WIN_LR_LOADFROMFILE )
 
-/* nSource: 0 ResourceIdByNumber
+/* https://msdn.microsoft.com/en-us/library/windows/desktop/ms648045(v=vs.85).aspx
```

Comment:
Minor nit: can you clean the above URL to:
https://msdn.microsoft.com/en-us/library/windows/desktop/ms648045.aspx

BTW, we're working again in master and there are 4 commits under it. Could you create a separate branch?